### ISSUE TITLE
Search functionality

### ISSUE THREAD

**Issue by ederrengifo** [ableco/politicalhub-client#22]
We need to implement a functionality to search among Candidatos and Partidos from the header.

**QUESTIONS TO SOLVE**
- What is the best technical approach to address this challenge?

**Comment by jvalenciag** [comment]
I was thinking in using some full search text engine like elasticsearch or postgres and feed all information we have available so  we can search for more that the name of a given candidate. what you guys think? @ederrengifo @hpneo

**Comment by hpneo** [comment]
I agree, but I also think it's going to depend on which fields are searchable and if the search needs to be as powerful to enable something like elasticsearch (vs a _simple_ `LIKE %` for example).
